conf/modules.config

```diff
@@ -152,9 +348,54 @@ process {
                 params.layout_algorithm ? "--layout-algorithm ${params.layout_algorithm} " : '',
             ].join(' ').trim()
         }
+
+        publishDir = [
+            [
+                path: { "${params.outdir}/pixelator" },
+                mode: params.publish_dir_mode,
+                pattern: 'layout/*.dataset.pxl',
+                saveAs: {
+                    // Trim the annotate directory prefix from the output name
+                    new File(it).name
```

Comment:
This saveAs function will only be run for the layout step on files that match the pattern. So whenever we have layout datasets we need to publish them to the output dir. There are no checks here like with the other dataset publishing since this is the "final" dataset.